braindecode/classifier.py

```diff
@@ -59,13 +59,15 @@ class name. Name conflicts are resolved by appending a count suffix
     __doc__ = update_estimator_docstring(NeuralNetClassifier, doc)
 
     def __init__(self, *args, cropped=False, callbacks=None,
```

Comment:
```suggestion
    def __init__(self, *args, *, cropped=False, callbacks=None,
```

I think we should enforce this but maybe in another PR

Reply:
Hi @agramfort,

I find this parameter change very interesting, but I agree with you that this has to be addressed in another pull request. I think that with the proposed code change at the moment, we maybe already have a change in several places in the library, or not.

Reply:
👌